pcsx2/gui/AppCoreThread.cpp

```diff
@@ -376,23 +389,26 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
 				gameMemCardFilter = game.getString("MemCardFilter");
 			}
 
-			if (EmuConfig.EnablePatches) {
-				if (int patches = InitPatches(gameCRC, game)) {
+			if (EmuConfig.EnablePatches)
+			{
+				int patches = InitPatches(gameCRC, game);
+				int fixes = loadGameSettings(fixup, game);
+
+				if (patches)
+				{
 					gamePatch.Printf(L" [%d Patches]", patches);
 					PatchesCon->WriteLn(Color_Green, "(GameDB) Patches Loaded: %d", patches);
 				}
-				if (int fixes = loadGameSettings(fixup, game)) {
+				if (fixes)
 					gameFixes.Printf(L" [%d Fixes]", fixes);
```

Comment:
First link only deals with assignment. It does not apply when you declare the variable within the if statement condition. Second link is user applying first link stuff when unnecessary and causing a compile error.

Try `if (int a = 1) std::cout << "Hey";` and tell me if there are warnings.